src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -1494,14 +1494,7 @@ module Util =
             makeLibCall com ctx None "Seq" "ofArray" [ ar ]
 
         // casts to generic param
-        | _, Fable.GenericParam(name, _isMeasure, _constraints) ->
-            makeCall
-                [
-                    name
-                    "from"
-                ]
-                None
-                [ expr ] // e.g. T::from(value)
+        | _, Fable.GenericParam(name, _isMeasure, _constraints) -> makeCall (name :: "from" :: []) None [ expr ] // e.g. T::from(value)
```

Comment:
Just asking to learn!
Why did you make the decision to move from `[ name; from ]` to `(name :: "from" :: [])`?

Reply:
@zprobinson Just so the formatter doesn't split it into multiple lines.
I only do this where it makes sense, and it also resembles Rust path naming which uses `::`.

Reply:
gotcha! make sense to me.